ui/src/common/components/__tests__/EmptyOrChildren.test.jsx

```diff
@@ -94,4 +94,16 @@ describe('EmptyOrChildren', () => {
     );
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('does not render children and shows title and description', () => {
+    const data = null;
+    const title = 'title';
+    const description = 'description';
```

Comment:
put `<div>description</div>` to be more clear, incase somebody reads the test cases as a source of documentation.